### PR TITLE
Address all current `content: correction` issues 

### PR DESCRIPTION
**Pull request by mechanomi**
## Summary of the changes / Why this improves CrateDB

all commits on this PR are for https://github.com/crate/crate/issues/10773

## Checklist

 - [ ] Added an entry in `CHANGES.txt` for user facing changes
 - [ ] Updated documentation & `sql_features` table for user facing changes
 - [ ] Touched code is covered by tests
 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [ ] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

File: docs/concepts/joins.rst [hunk 1]
```diff
@@ -308,11 +308,11 @@ optimized.*
    Complex queries are broken down into subqueries that are run on their shards
    before joining.
 
-.. _this article: http://www.codeproject.com/Articles/33052/Visual-Representation-of-SQL-Joins
+.. _this article: https://www.codeproject.com/Articles/33052/Visual-Representation-of-SQL-Joins
 .. _table function: https://crate.io/docs/reference/en/latest/sql/table_functions.html
 .. _information_schema: https://crate.io/docs/reference/sql/information_schema.html
 .. _system tables: https://crate.io/docs/reference/sql/system.html
-.. _here: http://www.dcs.ed.ac.uk/home/tz/phd/thesis.pdf
+.. _here: https://www.dcs.ed.ac.uk/home/tz/phd/thesis.pdf
```
Comment (mkleen) [inline docs/concepts/joins.rst:315]: This link does not work with `https`

**Comment by mkleen** [comment]
Looks good, but could you please check on all the links if they still work with https ? I found directly a broken one.

**Review by mkleen — APPROVED** [review]
The changes look great, however i found directly one link which didn't work with `http`. Could you please check the links ?

**Comment by mechanomi** [comment]
currently blocked on https://github.com/crate/crate/issues/10876

**Pull request merged**